tests/i18n/tests.py

```diff
@@ -1849,3 +1849,22 @@ def test_failure_finding_default_mo_files(self):
         self.patchGettextFind()
         trans_real._translations = {}
         self.assertRaises(IOError, activate, 'en')
+
+
+class NonDjangoLanguageTests(SimpleTestCase):
+    """
+    Test that a language non present in default Django languages can still be
```

Comment:
I prefer stating the expected behavior rather than "Test that..." since all tests are for testing. e.g. "A language not present in settings.LANGUAGES can be installed/used by a project."